dnase/parseSamtoolsGenotypesToBedFiles.pl

```diff
@@ -166,6 +175,20 @@ sub parseGenotype {
         if (($GT1 ne $GT2) or ($GQ1 != $GQ2)) {
             die "Wrong parsing assumptions for VCF genotype ($genotype)\n";
         }
+    } elsif ($format eq "GT:GL:GQ:FT:RCL:RC:RCR:RDCN:DR:DV:RR:RV") {
+        my ($GT1,$GL1,$GQ1,$FT1,$RCL1,$RC1,$RCR1,$RDCN1,$DR1,$DV1,$RR1,$RV1) = split /\:/, $genotype;
```

Comment:
I think the 1/2 suffices are added above to deal with some strings where the first and second allele are present as two entries. I'm not sure every clause follows this, but we at least shouldn't make it any worse.

Since, I don't think that any of the DELLY fields are specific to a single allele, these variables should be named "$GT,$GL,$GQ,$FT,$RCL,$RC,$RCR,$RDCN,$DR,$DV,$RR,$RV"

Reply:
I will update this

Reply:
$GT is defined outside of the if tree `my ($GT, $PL, $qualityScore, $readDepth);`.

Removing the number results in an error
```Use of uninitialized value $GT in split at ../../../../scripts/mapping/dnase/parseSamtoolsGenotypesToBedFiles.pl line 199, <IN> line 179.```

There needs to be a distinction between the inner scope variable and the outer.

Reply:
You said your test code had worked -- is this still the latest code?

Maybe I misinterpreted 1/2 as referring to alleles when really they are prefixes to avoid collisions between the two scopes

Reply:
I reverted back to the numbered version after I got the error.
This is the latest code